mmfunctions

```diff
@@ -148,6 +155,10 @@ _premis_event_list(){
     if [ "$eventDetail" = "makewaveform" ] ; then
         eventType="creation"
     fi
+    
+    if [ "$eventDetail" = "makefingerprint" ] ; then
+        eventType="creation"
```

Comment:
I think 'fixity' isn't in the PREMIS eventType vocab, they use 'message digest calculation' for the creation of a checksum, and 'fixity check' for the validation. But the vocab is limited.. http://id.loc.gov/vocabulary/preservation/eventType.html